samples/mongo/src/main/resources/static/resources/js/jquery/jquery.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v1.8.2 jquery.com | jquery.org/license */

```

Comment:
This sample is a clone of a boot sample. In boot-sample there is nice webapp to show the application usage and as I know it uses those resources.